### PR TITLE
feature-add avatar

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
Added avatar to component library

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    width: 100%;
+}
+
+.avatar.square {
+    border-radius: 2px;
+}
+
+.avatar-status {
+    border: 2px solid #fff;
+    background-color: #bcc3ce;
```
Comment (Hamdrive) [inline components/avatar/avatar.css]: consider using CSS variables instead

File: css/spacing.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.xs {
+    width: 8px;
+    height: 8px;
+}
+
+.sm {
+    width: 16px;
+    height: 16px;
```
Comment (Hamdrive) [inline css/spacing.css]: Can use rems for sm and beyond values

File: css/typography.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+h1, .xlg-para {
+    font-size: 48px;
+    line-height: 64px;
```
Comment (Hamdrive) [inline css/typography.css]: same here, use rem for large values

File: css/utilities.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.flex {
```
Comment (Hamdrive) [inline css/utilities.css]: Use a more specific name?
like `flex-row`

**Review by Hamdrive — COMMENTED** [review]
Looks good, have suggested some changes that can improve your code quality

**Review by GuptaEesh — APPROVED** [review]
Looks good to me. Just add a new line at EOF.

File: css/utilities.css (hunk 4 above)
Comment (theprogrammerinyou) [reply css/utilities.css]: Done.

File: css/typography.css (hunk 3 above)
Comment (theprogrammerinyou) [reply css/typography.css]: Done.

File: css/spacing.css (hunk 2 above)
Comment (theprogrammerinyou) [reply css/spacing.css]: Done.

File: components/avatar/avatar.css (hunk 1 above)
Comment (theprogrammerinyou) [reply components/avatar/avatar.css]: Done

**Review by Hamdrive — APPROVED** [review]
Great work done, changes have been successfully implemented!
Will suggest to look into how to write meaningful commit messages and PR descriptions
Else Good to merge!

**Pull request merged**